<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreambleSettings</class>
 <widget class="QDialog" name="PreambleSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New preamble</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="2">
    <widget class="MiniSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="MiniSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Switch" name="UseTemplate">
          <property name="text">
           <string>New preamble from template</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_5">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QRadioButton" name="PdfLatexButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>PdfLaTeX</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="XeLatexButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>XeLaTeX</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Document class</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="ClassCombo">
                <property name="editable">
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text">
                  <string>article</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>book</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>report</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>beamer</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>letter</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Font size</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="FontSizeCombo">
                <item>
                 <property name="text">
                  <string>10pt</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>11pt</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>12pt</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Paper size</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="PaperSizeCombo">
                <property name="editable">
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text">
                  <string>a4paper</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>a5parer</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>b5paper</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>letterpaper</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>executivepaper</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>legalpaper</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QCheckBox" name="InputCheck">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Input encoding</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="InputCombo"/>
              </item>
              <item>
               <widget class="QCheckBox" name="FontCheck">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Font encoding</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="FontEncCombo"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QCheckBox" name="BabelPolyglossiaCheckBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Babel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="BabelList"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_6">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTreeWidget" name="TamplateTree">
              <property name="columnCount">
               <number>2</number>
              </property>
              <column>
               <property name="text">
                <string>Template</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string notr="true">2</string>
               </property>
              </column>
              <item>
               <property name="text">
                <string>System</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Custom</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Packages</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="PackageFilter">
            <property name="placeholderText">
             <string>Search package...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="AddButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../images.qrc">
              <normaloff>:/images/go-next.svg</normaloff>:/images/go-next.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RemoveButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../images.qrc">
              <normaloff>:/images/go-previous.svg</normaloff>:/images/go-previous.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QListView" name="PackageList"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="LatexTextWidget" name="PreambleContentWidget"/>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="PackageDescription">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="OkButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Primary key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CodeLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NameLine"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LatexTextWidget</class>
   <extends>QTextEdit</extends>
   <header>latexeditorwidget.h</header>
  </customwidget>
  <customwidget>
   <class>MiniSplitter</class>
   <extends>QSplitter</extends>
   <header>minisplitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Switch</class>
   <extends>QCheckBox</extends>
   <header>switch.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
